docs/training/technology/recognition.md

```diff
@@ -37,31 +37,21 @@ We use this YAML markdown template for each team member.
 
 **Notes:**
 ```{warning}
-You must precisely follow the indentation and spacing layout shown here, so that the website correctly interprets the YAML format. Using the template is a great way to guarantee this.
+So that the website correctly interprets the YAML format we recommend using our template and not changing indent or spacing.
 ```
 - We encourage everyone to use their GitHub avatar as the source of their personal photograph.
 - Provide your GitHub username, together with the full url, for the `link` key under `social`.
-- Acceptable user groups (roles) can be found [here](https://uwhackweek.github.io/hackweeks-as-a-service/services/index.html).
+- Acceptable user groups (roles) can be found [here](https://escience.washington.edu/using-data-science/hackweeks/planning/hackweek-roles).
 
 ### Uploading to the website
 
-To include your content on the website, follow [these instructions](https://github.com/uwhackweek/jupyterbook-template/blob/main/team/README.md) (included below):
-* Navigate to the 'landing page' repository for your event.
-* Navigate into the 'teams' directory.
-* Use the provided 'template.yaml' file.
-* Create a copy of the 'template.yaml' file and rename the file to your name.
-You can do this through the GitHub interface, your local development environment, or on the command line with
-   ```
-    cp template.yaml FirstName-LastName.yaml
-   ```
-* Open the new file you just created in your favorite editor and fill out the
-  details, replacing the placeholder text.
-* Commit the file and (if necessary) push your changes to GitHub.
-* Open a pull request (PR).
-
-
-### UPDATE THESE IMAGES
-When you are just editing text, using GitHub's online text editor is very convenient:
+To include your content on the website, follow [these instructions](https://github.com/uwhackweek/jupyterbook-template/blob/main/team/README.md). Because YAML is a simple human-readable text format, you can add your content directly in the GitHub interface. We provide screenshots below for a particular event in 2024:

```

Comment:
Suggest: "We provide a screenshot below, for a past event, highlighting the GitHub edit button."